cvat/apps/engine/tests/test_rest_api_3D.py

```diff
@@ -208,6 +210,15 @@ def etree_to_dict(t):
 
 
 class Task3DTest(_DbTestBase):
+
+    def compare_annotations(self, a, b):
+        assert a.get('source') in ('manual', None)
+        assert b.get('source') in ('file', None)
```

Comment:
```suggestion
        assert b["source"] == "file"
```

I don't think it makes sense to allow `None` - if we want all loaded annotations to have source `"file"`, then that's what we should check.

Reply:
Makes sense, applying